3-networks-hub-and-spoke/shared.auto.example.tfvars

```
@@ -0,0 +1,19 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// List of IPv4 address of target name servers for the forwarding zone configuration.
+// See https://cloud.google.com/dns/docs/overview#dns-forwarding-zones
+target_name_server_addresses = ["192.168.0.1", "192.168.0.2"]
```

```suggestion
target_name_server_addresses = ["192.168.0.1", "192.168.0.2"]

//enable_hub_and_spoke_transitivity = true
```

missing the enable_hub_and_spoke_transitivity  variable in the right file example file